bentoml/_internal/service/service.py

```diff
@@ -90,6 +90,16 @@ def api(
         """Decorator for adding InferenceAPI to this service"""
 
         def decorator(func: t.Callable) -> t.Callable:
+            from ..io_descriptors import Multipart
+
+            if isinstance(output, Multipart):
+                logger.warning(
```

Comment:
this is probably not the best place to add it, move this to `InferenceAPI.__init__`?

Reply:
actually, let's just remove this? I think it is ok to just warn user about it in the documentation - otherwise this message will be all over the place in the logs if the user actually wants to use it